src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -10,7 +10,7 @@
 public abstract class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_USAGE = "clear person OR clear appt";
+    public static final String MESSAGE_USAGE = "clear person or appointment";
```

Comment:
Nitpick: the message usage should show the exact command format (`clear person` or `clear appt`) so that user knows exactly what command to type next. Also, it might be nice to add more detail, so something like:

![Screenshot 2024-10-21 at 10 06 07 PM](https://github.com/user-attachments/assets/777a65f6-fe72-402f-a5bd-2bd42fbc6695)